packages/client/rubygem-foreman_scap_client/foreman_scap_client-0.4.2.gem

```diff
@@ -0,0 +1 @@
+../../../.git/annex/objects/zf/21/SHA256E-s19968--fd4d7217891d11abc827f02f6527fed69fed805691dbc659a8008e5aebf4b46e.2.gem/SHA256E-s19968--fd4d7217891d11abc827f02f6527fed69fed805691dbc659a8008e5aebf4b46e.2.gem
```

Comment:
please remove the old `packages/client/rubygem-foreman_scap_client/foreman_scap_client-0.4.1.gem`, otherwise Jenkins is confused and fails.

Reply:
not sure how I missed that, removed now, thanks